ckanext/iati_generator/blueprint/iati.py

```diff
@@ -0,0 +1,11 @@
+from flask import Blueprint, render_template
+from ckan.plugins import toolkit
+
+iati_blueprint = Blueprint("iati_generator", __name__, url_prefix="/dataset")
+
+
+@iati_blueprint.route("/<id>/iati", methods=["GET"])
+def iati_page(id):
+    context = {"model": toolkit.model, "session": toolkit.model.Session, "user": toolkit.c.user}
+    toolkit.check_access("sysadmin", context)
+    return render_template("package/iati_page.html", pkg_id=id)
```

Comment:
```suggestion
@iati_blueprint.route("/<package_id>", methods=["GET"])
def iati_page(package_id):
    context = {"user": toolkit.c.user}
    toolkit.check_access("sysadmin", context)
    return render_template("package/iati_page.html", pkg_id=package_id)
```

`id` is a Python builtin function, it's better no to override it